specification/README.md

```diff
@@ -35,7 +35,24 @@ An implementation is not compliant if it fails to satisfy one or more of the "MU
 
 ## Document Statuses
 
-| Status               | Explanation                   |
-| -------------------- | ----------------------------- |
-| No Explicit "Status" | Equivalent to Experimental.   |
-| Experimental         | Breaking changes are allowed. |
+Documents and subsections within the specification are marked with statuses indicating their stability level. Functionality described in the specification graduates through these statuses with increasing stability. Stability levels apply only to normative sections within the specification; editorial changes to examples and explanations are exempt from these constraints.
+
+Possible statuses are described below:
+
+### Experimental
+
+Specification documents that are marked as `Experimental` contain functionality under active development. Breaking changes are allowed.
+
+### Hardening
+
+Documents marked as `Hardening` describe functionality with an emphasis on stabilizing existing requirements. Breaking changes require consensus by the [Technical Steering Committee](https://github.com/open-feature/community/blob/main/governance-charter.md#tsc-members).
+
+### Stable
+
+Sections marked as `Stable` do not allow breaking changes.
+
+### Mixed
+
+Specification documents marked as `Mixed` contain subsections of varying statuses specified above.
+
+\* No explicit status = `Experimental`
```

Comment:
is this an accident?

Reply:
No, I wanted it to show up as a literal `*`. Is there a better markdown symbol for this?

Reply:
I think you can do this:

> NOTE: No explicit status = `Experimental`